app/views/layouts/_breadcrumb.html.erb

```diff
@@ -1,17 +1,19 @@
-<div id="breadcrumb">
-	<%= link_to "Home", "/app#/teacher_sets" %> <i class="fa fa-angle-right"></i>
-	<% case request.fullpath %>
-	<% when "/account", "/users/edit" %>
-	Account
-	<% when "/help" %>
-	Help
-	<% when "/schools" %>
-	Participating schools
-	<% when "/users/start" %>
-  Get Started
-  <% when "/users/signup", "/users" %>
-  Sign Up
-	<% else %>
-	<%= request.fullpath %>
-	<% end %>
-</div>
+<% unless @hide_breadcrumbs %>
+	<div id="breadcrumb">
```

Comment:
What's the breadcrumbs functionality?

Reply:
@daryanypl it refers to the navigation links at the top-left of the page:
![screen shot 2018-08-29 at 9 36 33 am](https://user-images.githubusercontent.com/1825103/44791188-13ff0600-ab6f-11e8-82af-ebdb6c2a9813.png)